docs/source/advanced/backends.rst

```diff
@@ -1,14 +1,108 @@
+    | syncBlockKernels                                              | not necessary                                 | barrier                                                                       | barrier                                                                      | #pragma omp barrier                                                             | #pragma omp barrier                                                                                                                | __syncthreads                                    |
+    +---------------------------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+    | atomicOp                                                      | not necessary                                 | std::lockguard< std::mutex >                                                  | n/a                                                                          | #pragma omp critical                                                            | #pragma omp critical                                                                                                               | atomicXXX                                        |
+    +---------------------------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+    | ``ALPAKA_FN_HOST_ACC``, ``ALPAKA_FN_ACC``, ``ALPAKA_FN_HOST`` | inline                                        | inline                                                                        | inline                                                                       | inline                                                                          | inline                                                                                                                             | __device__, host, forceinline                    |
```

Comment:
`__device__` is now fixed, but the `_` are still missing in `host` and `forceinline`